addons/sourcemod/scripting/include/shavit/sql-create-tables-and-migrations.sp

```diff
@@ -508,6 +527,75 @@ void ApplyMigration_NormalizeMapzonePoints() // TODO: test with sqlite lol
 	QueryLog(gH_SQL, SQL_TableMigrationSingleQuery_Callback, sQuery, Migration_NormalizeMapzonePoints, DBPrio_High);
 }
 
+void ApplyMigration_NormalizeStageTimes()
+{
+	char sQuery[192];
+	// Check if stagetimeswr exists before migration (it does not on fresh installs)
+	if (gI_Driver == Driver_mysql)
+	{
+		FormatEx(sQuery, sizeof(sQuery), "SHOW TABLES LIKE '%sstagetimeswr';", gS_SQLPrefix);
+	}
+	else if (gI_Driver == Driver_sqlite)
+	{
+		FormatEx(sQuery, sizeof(sQuery), "SELECT 1 FROM sqlite_master WHERE type='table' AND name='%sstagetimeswr';", gS_SQLPrefix);
+	}
+	else // PostgreSQL unaffected
+	{
+		InsertMigration(Migration_NormalizeStageTimes);
+	}
```

Comment:
I think a `return;` should go right after the `InsertMigration` for postgres here. Does that seem right?